conf/alluxio-env.sh.template

```diff
@@ -1,67 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
@lijiang2 can you revert the change for this file?
Your PR should not delete this template file.
You can run `git checkout master conf/alluxio-env.sh.template conf/alluxio-site.properties.template` to recover these two files from the master branch